libraries/engage/cart/components/CartItems/CartItemsSubstitution.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { i18n } from '@shopgate/engage/core';
+import { CardList } from '@shopgate/engage/components';
+import Substitution, { SubstitutionWrapper } from '../Substitution';
+import connect from './CartItemsSubscription.connector';
+
+/**
+ * Renders the cart reservation card label.
+ * @param {Object} props The component props.
+ * @returns {JSX}
+ */
+const CartItemsSubstitution = ({
+  cartItems, updateProductsInCart, className, wrapCard,
+}) => {
+  const allAllowed = useMemo(() => cartItems.every(item => item.substitutionAllowed), [cartItems]);
+  const handleChange = useCallback(() => updateProductsInCart(cartItems.map(item => ({
+    cartItemId: item.id,
+    substitutionAllowed: !allAllowed,
+  }))), [allAllowed, cartItems, updateProductsInCart]);
+  if (wrapCard) {
+    return (
+      <SubstitutionWrapper>
+        <CardList.Item className={className}>
+          <Substitution
+            id="substitution-all"
+            label={i18n.text('cart.allow_substitution_all')}
```

Comment:
Spacing is not equal on small screens and the background color of the switch might not be optimal in every case.

![Bildschirmfoto 2020-07-23 um 18 45 07](https://user-images.githubusercontent.com/13087792/88314198-dc764580-cd14-11ea-8044-48ce79450728.png)